bubble.rb

```diff
@@ -0,0 +1,58 @@
+def bubble_sort(array)
```

Comment:
* [ ] bubble_sort method is implemented and working ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_bubble_sort/sort.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-1))
 Currently, ```bubble_sort``` is returning nil, hence any example i try also returns nil
```
int_array = [7, 5, 8, 3, 4, 6, 2, 4, 15, 2, 23]
p bubble_sort(int_array) = nil
```
Let us fix this